gen/http/oas2/generator.go

```diff
@@ -303,7 +303,10 @@ func (g *Generator) Generate(pkgInfo *generator.PkgInfo, spec *openapi.Specifica
 					name = "-"
 				}
 
-				return fmt.Sprintf("`%s:\"%s\"`", g.opts.SchemaTag, name)
+				if description == `` && !required {
+					return fmt.Sprintf("`%s:\"%s\"`", g.opts.SchemaTag, name)
+				}
+				return fmt.Sprintf("`%s:\"%s\" kun:\"descr=%s required=%v\"`", g.opts.SchemaTag, name, description, required)
```

Comment:
It's better to add `descr=?` tag option only when description is non-empty, and the same rule applies to `required=?`.